Index: BuilderPattern/obj/rider.project.restore.info
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>16878716244560377
===================================================================
diff --git a/BuilderPattern/obj/rider.project.restore.info b/BuilderPattern/obj/rider.project.restore.info
--- a/BuilderPattern/obj/rider.project.restore.info	
+++ b/BuilderPattern/obj/rider.project.restore.info	
@@ -1,1 +1,1 @@
-16878716244560377
\ No newline at end of file
+16878796452106573
\ No newline at end of file
